fastavro/_timezone.py

```diff
@@ -0,0 +1,49 @@
+"""Implementation of abstract base class tzinfo"""
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import time
+from datetime import datetime
+from datetime import timedelta
+from datetime import tzinfo
+
+"""Inspired by https://github.com/apache/avro/pull/207
+Obsoleted by datetime.timezone in py3.2"""
```

Comment:
To make sure I'm clear, if/when python 2 support is dropped, we should be able to remove `UTCTzinfo` and just use `datetime.timezone.utc`, correct?

Reply:
Correct. That's exactly right.